core/FrontController.php

```diff
@@ -494,14 +493,10 @@ protected function prepareDispatch($module, $action, $parameters)
 
     protected function handleMaintenanceMode()
     {
-        if ((GeneralConfig::getConfigValue('maintenance_mode') != 1) || Common::isPhpCliMode() ) {
```

Comment:
I think `GeneralConfig::getConfigValue` is the new function get General config. Maybe we should use `GeneralConfig::getConfigValue`

Reply:
Hm, I think maybe this change shouldn't have been committed, it's got nothing to do w/ the PR. I'll revert it.